### PR TITLE
Update Home page zero state when no active [sc-7538]

### PR DESCRIPTION
**Pull request by lvancraen**
Story: https://app.shortcut.com/capable/story/7538/update-home-page-zero-state-when-no-active-care-plan

Adds a filler in the currently blank home screen when a patient has no care plans.

@achidu for you to look at and say if there's anything to tweak 👇 

Screen Shot 2022-09-07 at 6 21 16 AM

### CODE REVIEW

**Comment by achidu** [comment]
@lvancraen looks great! One thing: today we have a shadow underneath the white header section at the top (see screenshot) — it makes it easier to distinguish between the header and the content below. 

Did we get rid of that at some point?

File: src/pages/Home.js [hunk 1]
```diff
@@ -96,7 +102,38 @@ const HomeContent = ({ carePlan }) => {
   const [tab, setTab] = useState(tabs[0]);
 
   if (!carePlan) {
-    return null;
+    const NoCarePlanCard = (text, textColor, left, top) => (
```
Comment (LUMacKenn) [inline src/pages/Home.js:105]: Can we make this a JSX component
```suggestion
    const NoCarePlanCard = ({text, textColor, left, top}) => (
```
and then render with those params as props

**Review by LUMacKenn — APPROVED** [review]
Looks good, just one minor thing

File: src/pages/Home.js (hunk 1 above)
Comment (lvancraen) [reply src/pages/Home.js]: Updated: [fbba127](https://github.com/capable-health/capable-care-demo/pull/31/commits/fbba127422c22f4e6131233f090165e99b713363)

**Pull request merged**